### PR TITLE
Turkish lang update

### PR DESCRIPTION
**Pull request by mesutustun**
Correcting the wrong translations in the Turkish language. more future

multilingual.txt
https://github.com/mesutustun/amxmodx/commit/d9c9611014cb40d1253efb581072003028277fe6

admincmd.txt
https://github.com/alliedmodders/amxmodx/commit/e32215846b55c539da253250aababada187944a8

plmenu.txt
https://github.com/mesutustun/amxmodx/commit/a460d0260489dae7eb2df8cf3b030dc5501b848d

adminhelp.txt
https://github.com/alliedmodders/amxmodx/commit/398f0dbc9058f77f0c22fea87a4a5f0544f94789

statsx.txt
https://github.com/alliedmodders/amxmodx/commit/403759424004a8fc223cf0c25bd1b90d887e96ce

statscfg.txt
https://github.com/alliedmodders/amxmodx/commit/15f6b4e47f8279c7505a8bde92429e18da94b34b

adminchat.txt
https://github.com/alliedmodders/amxmodx/commit/0243a46668915c9afb1cf17e2a7b6eeed8339408

adminvote.txt
https://github.com/alliedmodders/amxmodx/commit/9d08b1cff3dd6b3887181bfb622d84c8d38a192c

common.txt
https://github.com/alliedmodders/amxmodx/commit/8504b7d4d9617c6f74b98dfcb8d9579f5e04e9ae

menufront.txt
https://github.com/alliedmodders/amxmodx/commit/1397d4a3de67d8eca3d2c9bb0c4b17748cbe0602

miscstats.txt
https://github.com/alliedmodders/amxmodx/commit/cf65e92afe0da1b16f8de49dbd97c6d2faa6249d#diff-59a80bac894d4b4838a1cca063f1bb10

### CODE REVIEW

**Comment by eat1k** [comment]
42 commits...

File: plugins/lang/admincmd.txt [hunk 1]
```diff
@@ -226,79 +226,79 @@ ADMIN_EXTEND_2 = ADMIN %s: produziti mapu za %d minuta
...
+MAP_NOT_FOUND = Harita ismi bulunamadı yada mevcut değil
+ADMIN_MAP_1 = ADMIN: %s haritaya değiştirdi
+ADMIN_MAP_2 = ADMIN %s: %s haritaya değiştirdi
+NO_MORE_CVARS = Rcon'a daha cvar ekleyemiyorsunuz!
 UNKNOWN_CVAR = Bilinmeyen cvar: %s
 UNKNOWN_XVAR = Bilinmeyen xvar: %s
-CVAR_NO_ACC = O cvar'a giris hakkiniz yok
-XVAR_NO_ACC = O xvar'a giris hakkiniz yok
+CVAR_NO_ACC = O cvar'a erisiminiz yok
+CVAR_NO_ACC = O xvar'a erisiminiz yok
```
Comment (OciXCrom) [inline plugins/lang/admincmd.txt]: You changed the key name as well. The second one should be `XVAR_NO_ACC`, not `CVAR_NO_ACC`.

File: plugins/lang/admincmd.txt [hunk 2]
```diff
@@ -226,79 +226,79 @@ ADMIN_EXTEND_2 = ADMIN %s: produziti mapu za %d minuta
...
+ADMIN_KICK_1 = ADMIN: %s attı 
+ADMIN_KICK_2 = ADMIN %s: %s attı
+IP_REMOVED = Ip "%s" yasak listeden silindi
+AUTHID_REMOVED = Hüviyetli "%s" yasak listeden silindi
+ADMIN_UNBAN_1 = ADMIN: %s yasağını kaldırdı
+ADMIN_UNBAN_2 = ADMIN %s: %s yasağını kaldırdı
+ADMIN_ADDBAN_1 = ADMIN: %s yasakladı
+ADMIN_ADDBAN_2 = ADMIN %s: %s yasakladı
+BANNED = men'edildi
+REAON = Sebep
```
Comment (OciXCrom) [inline plugins/lang/admincmd.txt]: REAON => REASON. Be careful with the key names.

File: plugins/lang/statscfg.txt [hunk 3]
```diff
@@ -320,15 +320,7 @@ ST_SAY_REPORT = Say /report
...
 ST_SAY_STATS = Say /stats
-ST_SPEC_RANK = Spec. Rank Info
-ST_BOMB_PLANTED_SOUND = Bomb Planted Sound
-ST_BOMB_DEF_SUCC_SOUND = Bomb Defuse Succ. Sound
-ST_BOMB_DEF_FAIL_SOUND = Bomb Def. Failure Sound
-ST_BOMB_CD_HUD = Bomb Countdown Hud
-ST_LAST_MAN_SOUND = Last Man Sound
-ST_LAST_MAN_HEALTH = Last Man Health
-ST_HE_KILL_SOUND = Grenade Kill Sound
-ST_HE_SUICIDE_SOUND = Grenade Suicide Sound
```
Comment (OciXCrom) [inline plugins/lang/statscfg.txt]: Why did you remove these?

File: plugins/lang/admincmd.txt (hunk 1 above)
Comment (mesutustun) [reply plugins/lang/admincmd.txt]: I did not see

File: plugins/lang/admincmd.txt (hunk 2 above)
Comment (mesutustun) [reply plugins/lang/admincmd.txt]: i'll fix it right away

File: plugins/lang/statscfg.txt (hunk 3 above)
Comment (mesutustun) [reply plugins/lang/statscfg.txt]: https://github.com/alliedmodders/amxmodx/commit/01d20ee7f54d4dd86f53f192402c0ab8090ea04d

Comment (mesutustun) [reply plugins/lang/statscfg.txt]: > Why did you remove these?

https://github.com/alliedmodders/amxmodx/commit/01d20ee7f54d4dd86f53f192402c0ab8090ea04d

File: plugins/lang/admincmd.txt (hunk 2 above)
Comment (mesutustun) [reply plugins/lang/admincmd.txt]: > REAON => REASON.
> Be careful with the key names.

https://github.com/alliedmodders/amxmodx/pull/699/files#diff-5f49330f9b8a98d7f69ff555c9e5319c

**Comment by Arkshine** [comment]
What is the status of this PR?

**Comment by Arkshine** [comment]
Closing this due to lack of activity. Feel free to reopen it.
